### PR TITLE
Changed recommended nfs export options

### PR DESCRIPTION
**Pull request by miminar**
This aims to prevent from registry push failures.

See the bugzilla https://bugzilla.redhat.com/show_bug.cgi?id=1277356 for
details.

### CODE REVIEW

**Comment by miminar** [comment]
@stevedd, @pweil- may kindly I ask you for a review.

File: install_config/install/docker_registry.adoc [hunk 1]
```diff
@@ -957,17 +946,35 @@ During a push of an image, you may see one of the following errors:
...
 layer size can all contribute to potential errors that might occur when pushing
 an image using the default round-robin load balancing configuration.
 
-link:#session-affinity-workaround[One possible workaround] is to set the
-*docker-registry* service's `*sessionAffinity*` to *ClientIP*. This ensures that
-requests from a particular Docker client will always be handled by the same
-replica.
+There are two simple steps you can do to minimize probability of such a failure.
+
+1. Ensure that a `*sessionAffinity*` of your `docker-registry` service is set to `*ClientIP*`:
```
Comment (pweil-) [inline install_config/install/docker_registry.adoc]: s/a/the

File: install_config/install/docker_registry.adoc [hunk 2]
```diff
@@ -957,17 +946,35 @@ During a push of an image, you may see one of the following errors:
...
+Should give you `*ClientIP*` which is the default in recent OpenShift versions.
+If not, change it like this:
++
+----
+$ oc get -o yaml svc/docker-registry | \
+      sed 's/\(sessionAffinity:\s*\).*/\1ClientIP/' | \
+      oc replace -f -
+----
++
+2. Ensure that the NFS export line of you registry volume on your NFS server
```
Comment (pweil-) [inline install_config/install/docker_registry.adoc]: your

File: install_config/persistent_storage/persistent_storage_nfs.adoc [hunk 3]
```diff
@@ -318,8 +318,13 @@ exported volume on the NFS server should conform to the following conditions:
 - Each export must be:
 +
 ----
-/<example_fs> *(rw,root_squash)
+/<example_fs> *(rw,root_squash,sync,no_wdelay)
 ----
++
+Where the `sync` option causes server to commit all the changes to the
+underlying storage before sending a reply. And `no_wdelay` stops the server
```
Comment (pweil-) [inline install_config/persistent_storage/persistent_storage_nfs.adoc]: The `no_wdelay` option

File: install_config/persistent_storage/persistent_storage_nfs.adoc [hunk 4]
```diff
@@ -318,8 +318,13 @@ exported volume on the NFS server should conform to the following conditions:
...
+from delaying the writes. They proved to be necessary for correct functioning
```
Comment (pweil-) [inline install_config/persistent_storage/persistent_storage_nfs.adoc]: remove the "They..." sentence.

Comment (stevedd) [inline install_config/persistent_storage/persistent_storage_nfs.adoc]: On 04/15/2016 02:43 PM, Paul Weil wrote:
> In install_config/persistent_storage/persistent_storage_nfs.adoc <https://github.com/openshift/openshift-docs/pull/1908#discussion_r59921244>:
> 
>>  ----
>> ++
>> +Where the `sync` option causes server to commit all the changes to the
>> +underlying storage before sending a reply. And `no_wdelay` stops the server
>> +from delaying the writes. They proved to be necessary for correct functioning
> 
> remove the "They..." sentence.
I don't understand why the `sync` is necessary... Doesn't just `no_wdelay` work?

steved.

Comment (miminar) [inline install_config/persistent_storage/persistent_storage_nfs.adoc]: `no_wdelay` works only with `sync`. But `sync` is the default. So it should be fine to omit it. I'll do the change.

**Comment by miminar** [comment]
Items addressed.

**Comment by miminar** [comment]
@adellape this is ready for your sign-off.

**Comment by adellape** [comment]
@miminar I've pulled this commit into https://github.com/openshift/openshift-docs/pull/1935 and continued with edits there to merge at the same time. PTAL there. :bow:

**Comment by miminar** [comment]
Closing this in favor of #1935. Thanks @adellape!

**Pull request closed without merging**